pg_backup_api/pg_backup_api/spec/pg_backup_api.yaml

```diff
@@ -442,7 +442,7 @@ components:
           begin_time:
             description: >
               The system time of the Barman server at the start of the backup (in
-              ctime format in the local timezone of the server)
+              ISO 8601 in the local timezone of the server)
             nullable: true
             type: string
```

Comment:
I don't know that we need it, but type string also has the format option date-time which, if I'm understanding the RFC link, is the ISO 8601 format we're moving to. 

https://swagger.io/specification/#data-types

Reply:
Indeed, I was thinking using either date-time type if it suite, but I havn't checked yet its definition. It seems it will be suitable.
Backup plan was to use regex ...

Reply:
What may be limiting is that it does not take microseconds into account. This might be limiting for monitoring application. but maybe not. Might be worth clarifying it.